app/javascript/src/web-components/editable-content.js

```diff
@@ -206,6 +245,23 @@ class EditableContent extends HTMLElement {
     button.insertAdjacentHTML('beforebegin', deleteInputHTML);
     this.remove()
   }
+
+  // Run the markdown through all configured filter functions
+  // called by marked in the preprocess hook 
+  // called in the value() getter
+  filterMarkdown(markdown) {
+    for( const [key, filterFunction] of Object.entries(this.markdownFilters)) {
+      markdown = filterFunction(markdown)
+    }
+    return markdown
+  }
+
+  sanitize(unsafeHTML) {
+    return DOMPurify.sanitize(unsafeHTML, {
+        USE_PROFILES: {html: true}, 
+        FORBID_TAGS: ['style', 'button']
```

Comment:
Not 100% sure I am not suggesting silliness here but could we also add 'script' to the FORBID_TAGS for added sanitization goodness?

Reply:
I'll read the docs again, but I think `<script>` is blocked by default as that is pretty much the amin thing you want to do with an html sanitizer.  TBF, we don't actually need the `<button>` tag in there anymore - that was my first attempt at filtering out buttons, but I forgot they're not actually button elements, just styled links.

Reply:
Yep.  The profile 'html' includes a whole host of sanitization / xss prevention including removing `<script>` tags.  The `FORBID_TAGS` is then additive on top of that, specifying extra things to filter out.